<?xml version="1.0"?>
<launch>

    <!-- Load parameters -->
    <rosparam command="load" ns="" file="$(find grasping_pipeline)/config/config.yaml"/>
    <rosparam command="load" ns="" file="$(find table_plane_extractor)/config/config.yaml"/>
    <include file="$(find haf_grasping)/launch/haf_grasping_all.launch"/>
    <include file="$(find table_plane_extractor)/launch/get_objects_on_table_as.launch"/>
    <node name="table_plane_extractor_server" pkg="table_plane_extractor" type="table_plane_extractor_srv.py"/>
    <node name="execute_grasp_server" pkg="grasping_pipeline" type="execute_grasp_action_server.py" output="screen"/>
    <node name="handover_server" pkg="handover" type="handover_server.py"/>
    <node name="grasping_statemachine" pkg="grasping_pipeline" type="statemachine.py"  output="screen"/>
    <node name="placement" pkg="grasping_pipeline" type="place.py"  output="screen"/>
</launch>
